tests/suites/test_suite_ctr_drbg.data

```diff
@@ -1,723 +1,1075 @@
+CTR_DRBG_withDF.pdf: AES-256, PR=no, perso=no, add=no
+depends_on:MBEDTLS_CTR_DRBG_KEY_SIZE_256
```

Comment:
You need to update `MBEDTLS_CTR_DRBG_KEY_SIZE_256` to `!MBEDTLS_CTR_DRBG_USE_128_BIT_KEY` and `MBEDTLS_CTR_DRBG_KEY_SIZE_128` to `MBEDTLS_CTR_DRBG_USE_128_BIT_KEY`.